sdks/java/container/boot.go

```diff
@@ -193,7 +195,35 @@ func main() {
 	} else {
 		args = append(args, jammAgentArgs)
 	}
+	// Apply meta options
+	const metaDir = "/opt/apache/beam/options"
+	metaOptions, err := LoadMetaOptions(metaDir)
+	javaOptions := BuildOptions(metaOptions)
+	// (1) Add custom jvm arguments: "-server -Xmx1324 -XXfoo .."
+	args = append(args, javaOptions.JavaArguments...)
```

Comment:
Is there any risk of duplicate args here? What would happen in that case/should we do some validation/warn or overwrite?

Reply:
There may be duplicate args, but there's a priority associated with the MetaOptions, so higher priority will override the lower